examples/tutorials/Part 12 - Train an Encrypted Neural Network on Encrypted Data.ipynb

```diff
@@ -96,12 +94,8 @@
     "target_enc = target.fix_precision().share(alice, bob, crypto_provider=james)\n",
     "model_enc = model.fix_precision().share(alice, bob, crypto_provider=james)\n",
     "\n",
-    "# For now, we also need to encode the learning rate but this will change in the near future\n",
-    "learning_rate_fp = (\n",
-    "    torch.Tensor([learning_rate])\n",
-    "    .fix_precision()\n",
-    "    .share(alice, bob, crypto_provider=james)\n",
-    ")"
+    "# The learning rate is also changed to fixed precision\n",
+    "learning_rate_fp = torch.tensor(learning_rate).fix_precision()"
```

Comment:
Just wondering, why  `.share(alice, bob, crypto_provider=james)` was removed?

Reply:
Because if some settings we don't want to hide the config used, to the learning rate should be public, ie just a normal fixed precision tensor